src/Components/Locations.js

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+const Locations = () => {
+  const [locations, setLocations] = useState([]);
+  const [showLocations, setShowLocations] = useState(false);
+
+  useEffect(() => {
+    fetchLocations();
+  }, []);
+
+  const fetchLocations = async () => {
+    try {
+      const res = await axios.get(`https://ghibliapi.herokuapp.com/locations`);
+      setLocations(res.data);
+    } catch (err) {
+      setLocations([]);
+    }
+  };
+
+  const handleClick = () =>
+    setShowLocations((prevShowLocations) => !prevShowLocations);
+
+  const createLi = locations.map((locationObj) => {
+        let climate = locationObj.climate;
+        let terrain = locationObj.terrain;
+        if(climate === 'TODO'){
```

Comment:
Try using a ternary on line 25 and 26. 
const climate = locationObj.climate === 'TODO' ? "Unknown" : locationObj.climate